### DISCUSSION TITLE
AIOps Opni Admin Dashboard UX Enhancement

### DISCUSSION THREAD

**Discussion by AmartC** [rancher/opni#1001] [Ideas]
*Admin UI page*:
1. AIOps page:
    * Add a subtab `Log Anomaly Detection` and move everything into it. Within AIOps page, explicitly explain that this is for log anomaly detection. instead of enable AIOps , the button should be enable Log anomaly detection.
    * move the checkbox of `use GPU` to the worklaod page. remove `runtime class` until its ready(this requires reintroduce GPU operator)
    * pretrained models: Have hovering popup for the pretrained models explaining what each does.
2. Workload page: re-work the workload insights page:
            - Add an option to the Opni admin dashboard page to select all the deployments overall. Add an option to the Opni admin dashboard page page to select all deployments within a namespace.
            - GPU status check.
            - a button/checkbox to enable workload insights. Provide information to notify user that they need a GPU to use this feature. Once its enabled, gpu-related AI services get installed automatically.
            - hide the deployments until enabled.
            - once model done trainning, add a link to the opensearch plugin for the insights.
    

*Opni dashboard plugin*
-  Within Opni dashboard plugin, currently, broken down by Workload Pod and Workload Namespace. Add a section to break it down by deployment since user selected deployments to be added to the watchlist and pod names don't always correlate with deployment names.

(future feature) add an overview of the selected watchlist. If nothing wrong then user don't need to visit Opensearch

**Comment by tybalex** [comment]
scenarios
1. a new start: no GPU installed 
- GPU status check : false. 
- Add text such as  `no GPU installed in the cluster, in order to enable this feature, please install a GPU.`
- not allowed to enable the workload insights feature
2. a GPU and plugin installed. 
- GPU status check : true.
- Add text such as  `detected a GPU installed, would you like to enable the feature to start ingest workload insights`
- able to enable the feature.
- once enabled:
        * show the deployments and log counts(existing feature).
        * update/clear watchlist(existing feature).
        * once model done training, add a link to the Opensearch plugin for the insights.
3. workload insights enabled but GPU removed. 
- GPU status check : false. 
- watchlist still exists, but no new models can be trained.
- text box specify that no GPU is available.

**Comment by tybalex** [comment]
<img width="1103" alt="Screen Shot 2023-01-31 at 1 41 07 PM" src="https://user-images.githubusercontent.com/4568163/215889897-bd54448f-24a8-4282-97c3-ab0c4f2f192d.png">
<img width="944" alt="Screen Shot 2023-01-31 at 1 41 33 PM" src="https://user-images.githubusercontent.com/4568163/215890010-ff8d9ab2-343c-47b1-b639-8838270d96f0.png">

@codyrancher

**Comment by sanjay920** [comment]
```
AIOps
	Log Anomaly
		Pretrained Models
		Workload Insights
	Metric Anomaly
		...
```
